src/polymer/polymer2-config.ts

```diff
@@ -21,6 +21,39 @@ import {JavaScriptDocument} from '../javascript/javascript-document';
 import {analyzeProperties} from './analyze-properties';
 import {ScannedPolymerProperty} from './polymer-element';
 
+export function getIsValue(node: estree.ClassDeclaration|
+                           estree.ClassExpression): string|undefined {
+  const possibleIsGetters = node.body.body.filter(
+      (n) => n.type === 'MethodDefinition' && n.static === true &&
+          n.kind === 'get' && getIdentifierName(n.key) === 'is');
+  const isGetter = possibleIsGetters.length === 1 && possibleIsGetters[0];
+  if (!isGetter) {
+    return undefined;
```

Comment:
TODO: explore warning if multiple and warning if no "is" is determinable?

It's not obvious what conventions we want to enforce and lint for here, or whether we've got enough info though so maybe this isn't actionable enough.